holmes/plugins/toolsets/redis.py

```
@@ -0,0 +1,484 @@
+            for key, value in keyspace_info.items():
+                # Check if the key represents a database (e.g., 'db0', 'db1')
+                if key.startswith("db"):
+                  try:
+                    # Convert comma-separated string to a dictionary
+                      parts = value.split(',')
+                      db_info = {}
+                      for part in parts:
+                          k, v = part.split('=')
+                          db_info[k] = int(v)  # Convert values to integers
+                      formatted_info[key] = db_info
+                  except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)